app/org/sagebionetworks/bridge/services/NotificationsService.java

```diff
@@ -168,6 +184,35 @@ public void sendNotificationToUser(StudyIdentifier studyId, String healthCode, N
                 SEMICOLON_SPACE_JOINER.join(errorMessages) + ".");
         }
     }
+    
+    public void sendSMSMessage(StudyIdentifier studyId, Phone phone, String message) {
+        checkNotNull(studyId);
+        checkNotNull(phone);
+        checkNotNull(message);
+        
+        // Limited to 140 characters (Java is UTF-16, so two bytes per character, assuming for now that 
+        // SNS converts these to ASCII, requires integration testing
+        if (message.length() > BridgeConstants.SMS_CHARACTER_LIMIT) {
+            throw new BridgeServiceException("SMS message cannot be longer than 140 UTF-8/ASCII characters.");
+        }
```

Comment:
It's 140 bytes in GSM, which is not a standard  encoding in the JDK, but it is a 7-bit-per-character encoding with oddities, so we can approximate a measure with ASCII. It might throw an error or send a split message if too long... which is probably acceptable as it should ultimately come out during testing.